types/src/proto/transaction.proto

```diff
@@ -11,19 +11,6 @@ import "proof.proto";
 import "transaction_info.proto";
 import "google/protobuf/wrappers.proto";
 
-// The code for the transaction to execute
-message Program {
-    bytes code = 1;
-    repeated TransactionArgument arguments = 2;
-    repeated bytes modules = 3;
-}
-
-// The code for the transaction to execute
-message Script {
-    bytes code = 1;
-    repeated TransactionArgument arguments = 2;
-}
-
 // An argument to the transaction if the transaction takes arguments
 message TransactionArgument {
```

Comment:
Is this one still needed?

Reply:
I need to rewrite it in Rust as we use that in Canonical. I wanted to make this purely a deletion diff for review-ability. If I cannot get this second part done tomorrow, I'll create an issue.